src/app-layer-ftp.h

```diff
@@ -206,6 +206,7 @@ typedef struct FtpDataState_ {
     DetectEngineState *de_state;
     int32_t input_len;
     int16_t file_len;
+    int16_t file_mem_len;
```

Comment:
Is this ever something other than 0 or `file_len - 1`?

Reply:
0 and `file_len + 1`. I'll see if i can remove it.